tests/unit/converter/ena/test_ena_study_converter.py

```diff
@@ -0,0 +1,46 @@
+import json
+import unittest
+from os.path import dirname
+
+from assertpy import assert_that
+
+from converter.ena.ena_study import EnaStudyConverter
+
+
+class EnaStudyConverterTest(unittest.TestCase):
+
+    def setUp(self) -> None:
+        with open(dirname(__file__) + '/../../../resources/project.json') as file:
```

Comment:
could save the resources directory in a constant variable

Reply:
That is true. I am going to do it in my next PR, if that is ok with you. Thanks for this